python/packages/autogen-ext/src/autogen_ext/models/openai/_openai_client.py

```diff
@@ -213,6 +222,8 @@ def assistant_message_to_oai(
 def to_oai_type(message: LLMMessage) -> Sequence[ChatCompletionMessageParam]:
     if isinstance(message, SystemMessage):
```

Comment:
Check if model family is o1, if so then convert system message to developer message.